packages/components/addon/components/hds/pagination/nav/index.hbs

```diff
@@ -18,6 +18,22 @@
     </ul>
   {{/if}}
 
+  {{#if (eq this.type "truncated")}}
```

Comment:
I tried using `#unless` to avoid repeating these chunks of code but the linter complained it was too complex. (I searched but there doesn't seem to be a built in way to do a logical `or`)

Reply:
Let's sync on Monday, OK?